charts/tidb-cluster/values.yaml

```diff
@@ -31,12 +31,6 @@ pvReclaimPolicy: Retain
 # When enabling, scale in pd or tikv will trigger PV reclaim automatically
 enablePVReclaim: false
 
-# services is the service list to expose, default is ClusterIP
-# can be ClusterIP | NodePort | LoadBalancer
-services:
-  - name: pd
-    type: ClusterIP
-
```

Comment:
We should keep the old fields to keep the backward compatiability

Reply:
I think  the type config of `Services`  is working,the name is not working  because the name use hard code.
If we guarantee the type not changing, it will not trigger compatibility issues .

Reply:
That is, if the an user change the `ClusterIP` here to `NodePort`, and we remove these fields in the new version of helm chart, then the user's release would be broken on upgrading.

Reply:
General guideline is we could never delete public fields

Reply:
Okay,i know.